test/Microsoft.Azure.Relay.UnitTests/HybridRequestTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Relay.UnitTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Net.WebSockets;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Xunit;
+
+    public class HybridRequestTests : HybridConnectionTestBase
```

Comment:
@cesarme This file is new. It's very similar to the test cases you already reviewed elsewhere but it's ported to use xUnit and the rest of the test infrastructure here (e.g. TestUtility.Log and where to get a HC from).